src/server/server.ts

```diff
@@ -47,6 +47,15 @@ function isOfficeHost(host: string) {
     return officeHosts.indexOf(host) >= 0;
 }
 
+let assetPaths = null;
```

Comment:
You generally don't need to set to null.  Just leave it as its default undefined value, it's ok.